api/envoy/extensions/common/dynamic_forward_proxy/v3/dns_cache.proto

```diff
@@ -18,9 +18,21 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 
 // [#protodoc-title: Dynamic forward proxy common configuration]
 
+// Configuration of circuit breakers for resolver.
+message DnsCacheCircuitBreakers {
+  message DnsCacheThreshold {
```

Comment:
Sorry I still don't understand why we need this inner message?

/wait

Reply:
Should we use `config.cluster.v3.CircuitBreakers.Threshold`?

Reply:
Only to define `google.protobuf.UInt32Value` in `DnsCacheCircuitBreakers` directly?

Reply:
> Only to define google.protobuf.UInt32Value in DnsCacheCircuitBreakers directly?

That's what I would do.